batchflow/models/tf/deep_galerkin.py

```diff
@@ -311,58 +264,111 @@ def _build(self, config=None):
         self.output(output, predictions=config['predictions'], ops=config['output'], **config['common'])
 
     @classmethod
-    def ansatz(cls, inputs, **kwargs):
+    def body(cls, inputs, name='body', **kwargs):
+        """ Body of the neural network.
+        Shared between all of the unknown functions in the PDE."""
+        return super().body(inputs, name, **kwargs)
+
+    @classmethod
+    def head(cls, inputs, name='head', **kwargs):
+        """ Head (final) block of the neural network.
+        Makes one individual output branch for each unknown function in the PDE-system.
+        """
+        n_funs = kwargs.get('n_funs')
```

Comment:
```suggestion
        n_funs = kwargs.pop('n_funs')
```

Reply:
If there is no `n_funs` or it is not iterable, there will be a highly confusing error. Throw an exception